README.md

```diff
@@ -196,6 +196,101 @@ func main() {
 }
 ```
 
+### Create a Serverless Function
+```go
+package main
+
+import (
+	"fmt"
+	"github.com/twilio/twilio-go/twilio"
+	openapi "github.com/twilio/twilio-go/twilio/rest/serverless/v1"
+)
+
+func main() {
+	accountSid := os.Getenv("TWILIO_ACCOUNT_SID")
+	authToken := os.Getenv("TWILIO_AUTH_SID")
+	serviceSid := "ZSxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+	friendlyName := "My Serverless func"
+
+	client := twilio.NewClient(accountSid, authToken)
+
+	params := &openapi.CreateFunctionParams{
+		FriendlyName: &friendlyName,
+	}
+
+	resp, err := client.ServerlessV1.CreateFunction(serviceSid, params)
+	if err != nil {
+		fmt.Println(err.Error())
+		err = nil
+	} else {
+		fmt.Println(*resp.Sid)
+	}
+}
+```
+
+### Create a Studio Flow
+```go
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/twilio/twilio-go/framework/error"
+	"github.com/twilio/twilio-go/twilio"
+	openapi "github.com/twilio/twilio-go/twilio/rest/studio/v2"
+)
+
+func main() {
+	accountSid := os.Getenv("TWILIO_ACCOUNT_SID")
+	authToken := os.Getenv("TWILIO_AUTH_TOKEN")
+
+	commit := "commit"
+	friendlyName := "Studio flow from Go"
+	status := "draft"
+
+	var jsonStr = `{
+   "description":"Twilio Studio flow service",
+   "initial_state":"Trigger",
+   "states":[
+      {
+         "properties":{
+            "offset":{
+               "y":0,
+               "x":0
+            }
+         },
+         "transitions":[
+            
+         ],
+         "name":"Trigger",
+         "type":"trigger"
+      }
+   ]
+}`
+
+	definition := make(map[string]interface{})
+	err := json.Unmarshal([]byte(jsonStr), &definition)
+
+	client := twilio.NewClient(accountSid, authToken)
+	params := &openapi.CreateFlowParams{
+		CommitMessage: &commit,
+		Definition:    &definition,
+		FriendlyName:  &friendlyName,
+		Status:        &status,
+	}
+
+	resp, err := client.StudioV2.CreateFlow(params)
+	if err != nil {
+		fmt.Println(err.Error())
+		err = nil
+	} else {
+		fmt.Println(*resp.Sid)
+	}
+}
+```
+
```

Comment:
Since you are replacing `twilio "github.com/twilio/twilio-go/twilio"` in all of our other examples can you change the [handling exceptions](https://github.com/twilio/twilio-go/blob/update-readme/README.md#handling-exceptions) import statements to follow the same pattern?